containers/vsonline-linux/test-project/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+cd $(dirname "$0")
+
+# -- Utility functions --
+if [ -z $HOME ]; then
+    HOME="/root"
+fi
+
+FAILED=()
+
+check() {
+    LABEL=$1
+    shift
+    echo -e "\n🧪  Testing $LABEL: $@"
+    if $@; then 
+        echo "🏆  Passed!"
+    else
+        echo "💥  $LABEL check failed."
+        FAILED+=("$LABEL")
+    fi
+}
+
+checkMultiple() {
+    PASSED=0
+    LABEL="$1"
+    shift; MINIMUMPASSED=$1
+    shift; EXPRESSION="$1"
+    while [ "$EXPRESSION" != "" ]; do
+        if $EXPRESSION; then ((PASSED++)); fi
+        shift; EXPRESSION=$1
+    done
+    check "$LABEL" [ $PASSED -ge $MINIMUMPASSED ]
+}
+
+checkExtension() {
+    checkMultiple "$1" 1 "[ -d ""$HOME/.vscode-server/extensions/$1*"" ]" "[ -d ""$HOME/.vscode-server-insiders/extensions/$1*"" ]" "[ -d ""$HOME/.vscode-test-server/extensions/$1*"" ]"
+}
+
+# -- Actual tests - add more here --
+checkMultiple "vscode-server" 1 "[ -d ""$HOME/.vscode-server/bin"" ]" "[ -d ""$HOME/.vscode-server-insiders/bin"" ]" "[ -d ""$HOME/.vscode-test-server/bin"" ]"
+check "non-root-user" "id vsonline"
+check "/home/vsonline" [ -d "/home/vsonline" ]
+check "sudo" sudo -u vscode echo "sudo works."
+check "git" git --version
+check "command-line-tools" which top ip lsb_release
```

Comment:
There are a few other checks we currently run. Would be good to add those as well.

```
# Check platforms
docker run $IMAGE oryx platforms
docker run $IMAGE python --version
docker run $IMAGE dotnet --info
docker run $IMAGE node --version
docker run $IMAGE php --version

# Check expected shells
docker run $IMAGE bash --version
docker run $IMAGE fish --version
docker run $IMAGE zsh --version
```

Reply:
Good idea - added!